src/elisp/treemacs-mouse-interface.el

```diff
@@ -95,6 +110,12 @@ Clicking on icons will expand a file's tags, just like
         (funcall (cdr (assoc state treemacs-doubleclick-actions-config)))))
     (treemacs--evade-image)))
 
+(defun treemacs-dragleftclick-action (event)
+  (interactive "e")
+  (when (and (eq 'drag-mouse-1 (elt event 0)) treemacs-last-selected-button)
+    (select-window (elt (elt event 2) 0))
```

Comment:
The path is not checked for being a file.

Reply:
Yeah. Will do that.